index.js

```diff
@@ -286,6 +286,10 @@ function all(tree, file, config) {
    * @returns {boolean}
    */
   function irrelevant(word) {
-    return ignore.includes(word) || (ignoreDigits && /^\d+$/.test(word))
+    return (
+      ignore.includes(word) ||
+      (ignoreDigits && /^\d+$/.test(word)) ||
+      (ignoreDigits && /^\d{1,2}:\d{2}(?:[ap]\.?m)?$/i.test(word))
```

Comment:
```suggestion
      (ignoreDigits && /^\d{1,2}:\d{2}(?:[ap]\.?m\.?)?$/i.test(word))
```

I think I missed this dot earlier that might be there?